src/models/daily-dropoffs-report-data.test.ts

```diff
@@ -164,37 +165,52 @@ describe("DailyDropoffsReportData", () => {
   describe("#loadData", () => {
     const yearMonthDayParse = utcParse("%Y-%m-%d") as (s: string) => Date;
 
-    it("combines data across separate fetch requests", () => {
+    it("concatenates data across separate fetch requests", () => {
```

Comment:
luckily we already have tests for the aggregate behavior on its own: https://github.com/18F/identity-reporting/blob/2d014b9de3417974079e34c37aee2d3e43e22cb7/src/models/daily-dropoffs-report-data.test.ts#L15